src/index.js

```diff
@@ -137,6 +180,14 @@ function activateListeners() {
   activateCompletedListener();
 }
```

Comment:
The functions activateMoreListeners, activateCheckboxListeners, activateTaskInputListeners, and activateCompletedListener share similar patterns. Consider creating a generic function to reduce duplication.